app/jobs/slack_event/admins_job.rb

```diff
@@ -0,0 +1,71 @@
+#
+# This comes through the slash commands API which has a
+# slightly different payload than the events API
+#
+# Example payload:
+#
+# {
+#   "token": "xxxx",
+#   "team_id": "TCF0ZD3LL",
+#   "team_domain": "railslink-dev",
+#   "channel_id": "xxxx",
+#   "channel_name": "xxxx",
+#   "user_id": "xxxx",
+#   "user_name": "xxxx",
+#   "command": "/admins",
+#   "text": "",
+#   "response_url": "https://hooks.slack.com/commands/TCF0ZD3LL/640895425681/GDK2vHvOpxpvFrpRJnrvxiHS",
+#   "trigger_id": "xxxx",
+#   "slack": {
+#     "event": {
+#       "type": "admins"
+#     }
+#   },
+#   "controller": "slack",
+#   "action": "event"
+# }
+#
+
+class SlackEvent::AdminsJob < ApplicationJob
+
+  attr_reader :options
+  private :options
+
+  def perform(options = {})
+    @options = options.with_indifferent_access
+    message = {
+      text: admin_text,
+      response_type: "in_channel"
+    }
+    HTTParty.post(URI::parse(@options[:response_url]), body: message.to_json)
+  end
+
+  private
+
+  # Return the text message for the list of admins.
+  #
+  # The text message should not be wrapped as that will break Slack's automatic
+  # wrapping on presentation.
+  #
+  # Returns String (or nil)
+  def admin_text
+  <<~END_OF_TEXT
+    Admins and their contact information:
+    #{admin_string}
+    END_OF_TEXT
+  end
+
+  def admin_string
+    admins_string = ""
+    SlackUser.admins.shuffle.each do |admin|
```

Comment:
```suggestion
    SlackUser.admins.order('RANDOM()').each do |admin|
```
This keeps the business logic/random ordering in the query. It results in a sequential order, but it's still faster and IMHO better than doing the shuffle in the ruby to keep the.
That said, maybe this is unfavorable in the end.

Reply:
```
This keeps the business logic/random ordering in the query. It results in a sequential order, but it's still faster and IMHO better than doing the shuffle in the ruby
```
I'm really unclear about what you're saying here.  If it's a random order query, why would it result in a sequential order?

Reply:
Sequential order would mean that it needs to order each record 1 by 1 instead of being able to use any indices for the order, which is less effort than Ruby would use, but more effort than `.order(:id)` would use.

Reply:
Personally I don't think any gains in speed are going to matter since we're talking about a query of less than a few dozen users right now.